<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerMacroEditWidget</class>
 <widget class="QWidget" name="ControllerMacroEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Binds/Buttons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select the buttons which you want to trigger with this macro. All buttons are activated concurrently.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="bindList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Pressure</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>For buttons which are pressure sensitive, this slider controls how much force will be simulated when the macro is active.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="pressureLayout">
        <item>
         <widget class="QSlider" name="pressure">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pressureValue">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Select the trigger to activate this macro. This can be a single button, or combination of buttons (chord). Shift-click for multiple triggers.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="triggerToggle">
          <property name="text">
           <string>Press To Toggle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="InputBindingWidget" name="trigger">
        <property name="text">
         <string notr="true">PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="deadzoneLayout" stretch="0,1,0">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Deadzone:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="deadzone">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="deadzoneValue">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>100%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Frequency</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
        <item>
         <widget class="QLabel" name="frequencyText">
          <property name="text">
           <string>Macro will toggle every N frames.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setFrequency">
          <property name="text">
           <string>Set...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="increaseFrequency">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="decreateFrequency">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputBindingWidget</class>
   <extends>QPushButton</extends>
   <header>inputbindingwidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
